ggu/main.c

```diff
@@ -16,11 +16,313 @@
  * along with this program.  If not, see <http://www.gnu.org/licenses/>.
  */
 
+#include <stdbool.h>
+
+#include <rte_ip.h>
+#include <rte_udp.h>
+#include <rte_log.h>
+#include <rte_lcore.h>
+#include <rte_malloc.h>
+#include <rte_ethdev.h>
+
+#include "gatekeeper_gk.h"
 #include "gatekeeper_ggu.h"
+#include "gatekeeper_net.h"
+#include "gatekeeper_main.h"
+#include "gatekeeper_config.h"
+
+/* The UDP source and destination port numbers for GGU. */
+#define GGU_SRC_PORT (0xA0A0)
+#define GGU_DST_PORT (0XB0B0)
```

Comment:
They can be kept here as long as there's an XXX tag asking to make them available in ggu_conf.